src/openrct2/object/ObjectList.cpp

```diff
@@ -38,6 +38,8 @@ int32_t object_entry_group_counts[] = {
     MAX_TERRAIN_EDGE_OBJECTS,
     MAX_STATION_OBJECTS,
     MAX_MUSIC_OBJECTS,
+    MAX_FOOTPATH_SURFACE_OBJECTS,
+    MAX_FOOTPATH_RAILINGS_OBJECTS,
```

Comment:
Please include all the limits even if they are zero, this should reflect all limits combined.

Reply:
Uhm, this _does_ include all the limits...

Reply:
Sorry I got confused about the two tables, the other one was lacking a few.